### DISCUSSION TITLE
Error: Node with name 'xxxx' already registered.

### DISCUSSION THREAD

**Discussion by SnowfungusWang** [antvis/X6#3871] [Q&A]
请问有什么办法可以获取当前注册过的自定义节点列表吗？
在vue2.x中当想在一个页面中使用两个不同的图时，由于在每个图中分别注册了自定义节点custom-node，遇到了报错：
[Vue warn]: Error in created hook: "Error: Node with name 'custom-node' already registered."
![image](https://github.com/antvis/X6/assets/37967256/2af85968-558c-488c-8a73-79643e19a573)

**Comment by NewByVector** [comment]
传第三个参数 `true` 就像，每次注册的时候覆盖之前的。
